.github/workflows/dep.yml

```diff
@@ -57,3 +58,7 @@ jobs:
           -pl kyuubi-ctl,kyuubi-server,kyuubi-assembly -am
       - name: Check dependency list
         run: build/dependency.sh
+      - name: Dependency Review
+        uses: actions/dependency-review-action@v3
+        with:
+          fail-on-severity: low
```

Comment:
Is `low` too strict? Use moderate?

Reply:
The docs of action does not clarify how it classify the level of severity, so I prefer to keep it in `low` the same as the default value. We could lower this level if necessary in future, as it only affects future PRs.

Reply:
The risk for vulnerability is determined by the CVSS score. If you go to check the mvn central or NVD, it's common to see an artifact have some low-level vulnerabilities. I guess it is not practical for us to add such a critical rule that blocks PRs frequently

Reply:
OK, changed to `moderate`.